docs/business-finder-link-based-facets.md

```diff
@@ -0,0 +1,51 @@
+# EU Exit Business Readiness finder link-based facets
+
+This document explains why the [EU Exit Business Readiness Finder](https://www.gov.uk/find-eu-exit-guidance-business) ("business finder") uses [content-tagger](https://github.com/alphagov/content-tagger) 'links' to filter the finder.
+
+## How the business finder is different to other finders
+
+### From the user perspective
+
+In most finders, search results get more granular as you select values in different facets (the number of results goes down). The business finder search results get more _broad_ as you select values in different facets (number of results goes up).
+
+The business finder can be sorted by topic ("grouped display"). In this context, a 'topic' is the label of the facet (such as 'Personal data'), except for the 'Sector / Business area' facet, where the _facet value_ (such as 'Aerospace') is the topic.
+
+![Screenshot of 'Sort by Topic'](https://user-images.githubusercontent.com/93511/52811122-38eb2500-308c-11e9-843f-4df29b719235.png)
+
+This view only applicable to the business readiness finder, but is not yet a candidate for deletion, as it's been proven to be a useful feature.
+
+### From the content designer perspective
```

Comment:
I'd argue that this whole section should be moved to content-tagger as it describes the tagging process that happens there.

Reply:
Have raised https://github.com/alphagov/content-tagger/pull/944.

Reply:
Thanks! I'll go and take a look.